nilearn/maskers/nifti_masker.py

```diff
@@ -263,6 +274,7 @@ def __init__(
         memory=None,
         verbose=0,
         reports=True,
+        filter=None,
```

Comment:
no I don't think we want to do that, because if we start doing this then we we will end up passing all the possible arguments of image.clean to all our maskers

though admittedly we actually need to also clean up the maskers API to on the long run get rid of kwargs (but that's another issue)